xlators/features/trash/src/trash.c

```diff
@@ -2243,9 +2244,19 @@ reconfigure(xlator_t *this, dict_t *options)
     GF_OPTION_RECONF("trash-internal-op", priv->internal, options, bool, out);
     GF_OPTION_RECONF("trash-dir", tmp, options, str, out);
 
+    active_earlier = priv->state;
     GF_OPTION_RECONF("trash", priv->state, options, bool, out);
+    if (active_earlier && !priv->state) {
+        if (priv->trash_itable) {
+            inode_table_destroy(priv->trash_itable);
```

Comment:
Is it safe to do that ? how do we prevent inodes from being used while or after the table has been destroyed ?

Reply:
Done